### DISCUSSION TITLE
Possibility to reach the api from a backend plugin

### DISCUSSION THREAD

**Discussion by lahabana** [grafana/grafana#38087] [Plugins]
On the frontend plugins it's possible to do `getBackendSrv()` which doesn't seem possible on backend plugins.

This would enable building datasources that's enriched with grafana info or could even get enriched with existing datasource.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
